src/Core/Framework/App/Exception/ActionNotFoundException.php

```diff
@@ -7,7 +7,7 @@
 use Symfony\Component\HttpFoundation\Response;
 
 /**
- * @internal only for use by the app-system, will be considered internal from v6.4.0 onward
+ * @final only for use by the app-system, will be considered internal from v6.4.0 onward
```

Comment:
I'm not sure about the rule about exceptions here, IMHO there is also the case for saying they are internal and you can rely on the base class :thinking:

Reply:
especially in the app namespace all exceptions should be internal, as the whole module should stay internal